### ISSUE TITLE
Is there support for Shell.Resources

### ISSUE THREAD

**Issue by bryanp-ramosoil** [xamarin/MobileBlazorBindings#102]
Hello,

I had a quick question regarding Shell.Resources. It doesn't look the this is supported yet. Is there a work around for this? I'm trying to style my flyout menu and all my attempts have failed.

**Comment by Eilon** [comment]
Hi @bryanp-ramosoil you are correct it isn't supported at this time. Do you know if it's possible to style the flyout in other ways, such as with regular properties or with CSS? Or is Resources the only way?

**Comment by bryanp-ramosoil** [comment]
I've been trying to get it to work with style sheets but it's not working. The way Shell does it is within the ContentPage.Resources of the XAML.

<ContentPage.Resources>
        <StyleSheet Source="/Assets/styles.css" />
</ContentPage.Resources>

I've also tried 

using (var reader = new StringReader("^contentpage { background-color: lightgray; }"))
{
     this.Resources.Add(StyleSheet.FromReader(reader));
}

Both attempts have failed.

**Comment by Eilon** [comment]
Here's an example of how to use Stylesheet: https://github.com/xamarin/MobileBlazorBindings/blob/master/samples/MobileBlazorBindingsWeather/MobileBlazorBindingsWeather/WeatherApp.razor#L4

**Comment by bryanp-ramosoil** [comment]
Perfect thanks. I don't know if this is an issue but it only seems to work if the css file is the root of the application. I've attempted to put the file in an Assets folder then reference it using it's name space and it fails. I've used a decompiler to verify the resource was there and it was.

Fails
Example: MyProject.Assets.styles.css
Folder: root/Assets/styles.css

Works 
Example: styles.css
Folder: root/styles.css

It's not a deal breaker but I thought I would pass the information along. 

Thank you for creating this project. This is perfect for us web developers who now need to start building mobile apps.
